docs/proposals/instrumentation-of-services.md

```diff
@@ -0,0 +1,70 @@
+# Best practices for instrumenting infrastructure components in OpenShift
+
+All OpenShift components MUST expose the following endpoints on either their public HTTPS port, or a public HTTP port if none exists
+
+* `/healthz` - returns 200 and the text `ok` if the instance is alive
+* `/metrics` - returns a reasonable set of metrics in the Prometheus format for capturing the health and performance of the server in question
```

Comment:
That's why it's listed as auth.  But yes, if services have both internal and external, prefer internal only.  However, any metrics with tenant info need auth - if you have auth, internal vs external doesn't matter as much.  

Another problem is that if we do host networking we only get the hosts ip - would be better if we got a pod network ip as well in the future